osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerLoungeSubScreen.cs

```diff
@@ -21,6 +25,32 @@ public class MultiplayerLoungeSubScreen : LoungeSubScreen
         [Resolved]
         private MultiplayerClient client { get; set; }
 
+        private MultiplayerListingPollingComponent multiplayerListingPollingComponent => (MultiplayerListingPollingComponent)ListingPollingComponent;
+
+        private readonly IBindable<bool> isConnected = new Bindable<bool>();
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            isConnected.BindTo(client.IsConnected);
+            isConnected.BindValueChanged(c => Scheduler.AddOnce(() => multiplayerListingPollingComponent.AllowPolling = c.NewValue));
+            multiplayerListingPollingComponent.AllowPolling = isConnected.Value;
```

Comment:
@smoogipoo are you okay if i move this logic inside the `MultiplayerListingPollingComponent` implementation? Seems like it's a better home for it.

Reply:
Yep, sounds good